### PR TITLE
Pagerduty API alerts

### PR DESCRIPTION
**Pull request by benwarfield-usds**
## Related Issue or Background Info

- Resolves CDCGov/prime-devops#2

## Changes Proposed

- fix some minor issues about the alerts terraform module
- make stg and prod route alerts to the SimpleReport - Production service, and everything else to SimpleReport - Demo

## Additional Information

This is being deployed already, but we can obviously redeploy if we change things, that's the point.

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
I was planning to leave this to merge after the parent branch was merged, but this PR fixes the validation problems in that PR, so that seems like a less good plan.

File: ops/services/alerts/app_service_metrics/main.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ locals {
 
 resource "azurerm_monitor_metric_alert" "cpu_util" {
   name                = "${var.env}-api-cpu"
-  description         = "${local.ucfirst_env} CPU utilization is greater than 70%"
+  description         = "${local.ucfirst_env} CPU utilization is greater than ${var.cpu_threshold}%"
```
Comment (jdorothy) [inline ops/services/alerts/app_service_metrics/main.tf:7]: Nice!

File: ops/demo/alerts.tf [hunk 2]
```diff
@@ -3,8 +3,14 @@ module "metric_alerts" {
...
   app_service_id      = module.simple_report_api.app_service_id
-  action_group_id     = data.terraform_remote_state.global.outputs.slack_alert_action_id
   severity            = 1
   rg_name             = data.azurerm_resource_group.rg.name
   tags                = local.management_tags
+  mem_threshold       = 85
+
+  action_group_ids = [
+    data.terraform_remote_state.global.outputs.slack_alert_action_id,
+    data.terraform_remote_state.global.outputs.pagerduty_demo_action_id
```
Comment (jdorothy) [inline ops/demo/alerts.tf:13]: What is this - separate PR change?

Comment (benwarfield-usds) [reply ops/demo/alerts.tf:13]: Yes, the service code is in #1500. I think probably I should have done them both in one go, but I try to separate out the repetitive parts from the interesting parts.

**Pull request merged**